src/concepts/essentials/binary.md

```diff
@@ -0,0 +1,74 @@
+# Dealing with Binary Data
```

Comment:
I would create a new concept section `Backend runtime`

Reply:
"Dealing with Binary Data" is a lot more meaningful than "Backend runtime" for a user

Reply:
Section is the parent of the concept

Reply:
Ah gotcha, makes sense now 👌